### DISCUSSION TITLE
基础内置参数：${system.datetime} ，返回的不是上海的时间，而是-8小时的时间，这个如何调整啊？

### DISCUSSION THREAD

**Discussion by PeerlessHero** [apache/dolphinscheduler#14816] [Q&A]
基础内置参数：${system.datetime} ，返回的不是上海的时间，而是-8小时的时间，这个如何调整啊？

**Comment by PeerlessHero** [comment]
![image](https://github.com/apache/dolphinscheduler/assets/20982591/d3abeb24-69e0-4849-80db-6e6d68f5df45)
![image](https://github.com/apache/dolphinscheduler/assets/20982591/7e27719f-ffb0-4e8c-9022-90dc859df821)

当前时间是23点43，不是15点多，少了8个小时，这个如何调整啊？

**Comment by SharkAndCat** [comment]
你得自己调整时区，不然系统自己的时间就是UTC时区的，会慢8小时。把bin/env/dolphinscheduler_env.sh中的SPRING_JACKSON_TIME_ZONE=${SPRING_JACKSON_TIME_ZONE:-UTC} UTC修改为GMT+8就行了
